### DISCUSSION TITLE
Galera, quem usa node no servidor, usam Express, Nest, Hapi ou algum outro? Indicam?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1655] [Perguntas e Respostas]
Galera, quem usa node no servidor, usam Express, Nest, Hapi ou algum outro? 
Indicam algum? Contra indicam algum?

Abraços

**Comment by kivervinicius** [comment]
Pela facilidade e pelo "tempo de mercado" indico express, porém estou com api's rodando com o fastify  que se mostrou realmente ser bem mais rápido e com uma api bem semelhante. 

Da uma olhada no AdonisJS que é um framework completo mto bom também e é baseado no laravel do php caso já tenha tido experiência.
